### PR TITLE
Google.Terraformer version 0.8.24

### PR DESCRIPTION
**Pull request by qqii**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111512&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/Google/Terraformer/0.8.24/Google.Terraformer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: Google.Terraformer
+PackageVersion: 0.8.24
+Installers:
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://github.com/GoogleCloudPlatform/terraformer/releases/download/0.8.24/terraformer-all-windows-amd64
+  InstallerSha256: F69C09F7045A6010B892BF2F32229F3B5AE3F20F54FF975F2CBDEDCEE5CB0241
+  Commands:
+  - terraformer
```
Comment (qqii) [inline manifests/g/Google/Terraformer/0.8.24/Google.Terraformer.installer.yaml:12]: This doesn't seem to work?

Comment (Trenly) [reply manifests/g/Google/Terraformer/0.8.24/Google.Terraformer.installer.yaml:12]: It works for me. It probably has the same issue as #111454 where the base executable isn't renamed if symlinks aren't allowed

```
--> Installing the Manifest 0.8.24

Found Terraformer [Google.Terraformer] Version 0.8.24
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/GoogleCloudPlatform/terraformer/releases/download/0.8.24/terraformer-all-windows-amd64
  ██████████████████████████████   410 MB /  410 MB
Successfully verified installer hash
Starting package install...
Command line alias added: "terraformer"
Path environment variable modified; restart your shell to use the new value.
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName DisplayVersion Publisher ProductCode
----------- -------------- --------- -----------
Terraformer 0.8.24         Google    Google.Terraformer__DefaultSource



PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> terraformer
Usage:
   [command]

Available Commands:
  help        Help about any command
  import      Import current state to Terraform configuration
  plan        Plan to import current state to Terraform configuration
  version     Print the version number of Terraformer

Flags:
  -h, --help      help for this command
  -v, --version   version for this command

Use " [command] --help" for more information about a command.
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> terraformer -v
version v0.8.24
```

**Comment by Trenly** [comment]
[Policy] Reset Feedback

**Review by Trenly — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@qqii Please mark the above conversation as "resolved" for the auto-merge process to take effect

**Pull request merged**